MiscVBAFunctions/Modules/MiscDataContainers.bas

```diff
@@ -0,0 +1,61 @@
+Attribute VB_Name = "MiscDataContainers"
+Option Explicit
+
+
+Public Sub ConcatContainers(DataStructure1 As Variant, DataStructure2 As Variant)
```

Comment:
Not obvious from sub and argument names that DataStructure1 is added to? At least need to add some docstrings to explain what is being done here